t/22-svirt.t

```diff
@@ -162,11 +162,11 @@ subtest 'starting VMware console with Cloud Init' => sub {
         $s . ' destroy openQA-SUT-1 |& grep -v "\\(failed to get domain\\|Domain not found\\)"',
         $s . ' undefine --snapshots-metadata openQA-SUT-1 |& grep -v "\\(failed to get domain\\|Domain not found\\)"',
         $s . ' define /var/lib/libvirt/images/openQA-SUT-1.xml',
-        'echo bios.bootDelay = "10000" >> /vmfs/volumes/datastore1/openQA/openQA-SUT-1.vmx',
-        'echo guestinfo.metadata = "test@meta" >> /vmfs/volumes/datastore1/openQA/openQA-SUT-1.vmx',
-        'echo guestinfo.metadata.encoding = "gzip+base64" >> /vmfs/volumes/datastore1/openQA/openQA-SUT-1.vmx',
-        'echo guestinfo.userdata = "test%user" >> /vmfs/volumes/datastore1/openQA/openQA-SUT-1.vmx',
-        'echo guestinfo.userdata.encoding = "gzip+base64" >> /vmfs/volumes/datastore1/openQA/openQA-SUT-1.vmx',
+        'echo bios.bootDelay = "10000" >> /vmfs/volumes/' . $bmwqemu::vars{VMWARE_DATASTORE} . '/openQA/openQA-SUT-1.vmx',
+        'echo guestinfo.metadata = "test@meta" >> /vmfs/volumes/' . $bmwqemu::vars{VMWARE_DATASTORE} . '/openQA/openQA-SUT-1.vmx',
+        'echo guestinfo.metadata.encoding = "gzip+base64" >> /vmfs/volumes/' . $bmwqemu::vars{VMWARE_DATASTORE} . '/openQA/openQA-SUT-1.vmx',
+        'echo guestinfo.userdata = "test%user" >> /vmfs/volumes/' . $bmwqemu::vars{VMWARE_DATASTORE} . '/openQA/openQA-SUT-1.vmx',
+        'echo guestinfo.userdata.encoding = "gzip+base64" >> /vmfs/volumes/' . $bmwqemu::vars{VMWARE_DATASTORE} . '/openQA/openQA-SUT-1.vmx',
```

Comment:
I'm wondering whether all of these test changes are actually required. I'd rather avoid them considering that in the tests we use a defined set of `$bmwqemu::vars` values and therefore we can actually just keep the expected value here hard-coded (similar to how the name of the vmx file is also just hard-coded). That makes it easier to understand what the test actually expects here.

Reply:
OK. In fact, it will not impact testing on sle15sp5, will keep 'datastore1' as the default value.